deployer/src/deployer/search/models.py

```diff
@@ -51,14 +51,74 @@
     # With this configuration we'll get good matches for both
     # 'array foreach' and if people type the full 'array.prototype.foreach'.
     preserve_original=True,
-    # Otherwise things like 'WebGL' becomes 'web' and 'gl'.
-    # And 'forEach' becomes 'each' because 'for' is a stopword.
-    split_on_case_change=False,
+    # 'forEach' becomes 'for', 'each', and 'forEach'.
+    # And searchin for 'typed array' can find 'TypedArray'
+    split_on_case_change=True,
```

Comment:
reading the old comment I wonder: does this degrade "WebGL" searches?

Reply:
It would, yes! It will find documents that contain "Web" more. 
But I'll play with that.

Reply:
<img width="966" alt="Screen Shot 2021-05-10 at 2 00 32 PM" src="https://user-images.githubusercontent.com/26739/117704198-92255e80-b198-11eb-9d3d-e679759ebd30.png">
<img width="1053" alt="Screen Shot 2021-05-10 at 2 00 42 PM" src="https://user-images.githubusercontent.com/26739/117704210-96ea1280-b198-11eb-8636-752d9eaec2ae.png">



Compare these screenshots with:
https://developer.mozilla.org/en-US/search?q=webgl
https://developer.mozilla.org/en-US/search?q=web+gl

So, before, in `?q=web+gl`, it used to search for `"WEB" || "GL"` but now it's `"WEBGL" || "WEB" || "GL"`.